api/test/integration/mapping/integration_test_api_endpoints.json

```diff
@@ -3168,22 +3075,6 @@
             }
         ]
     },
-    {
-        "path": "api/test/integration/env/configurations/vulnerability/manager/configuration_files",
-        "files": [
-            {
-                "name": "nvd_providers.sh",
-                "tag": "nvd",
-                "tests": [
-                    "test_agent_DELETE_endpoints.tavern.yaml",
-                    "test_agent_GET_endpoints.tavern.yaml",
-                    "test_agent_POST_endpoints.tavern.yaml",
-                    "test_agent_PUT_endpoints.tavern.yaml",
-                    "test_cluster_endpoints.tavern.yaml"
-                ]
-            }
-        ]
-    },
     {
         "path": "api/test/integration/env/configurations/vulnerability/manager/healthcheck",
         "files": [
```

Comment:
If we removed `api/test/integration/env/configurations/vulnerability/manager/configuration_files`, shouldn't we delete this one as well?

Reply:
Your right, removed in a new commit.